toolkit/tools/imagecustomizer/docs/cli.md

```diff
@@ -0,0 +1,83 @@
+# Mariner Image Customizer command line
+
+## --help
+
+Displays the tool's quick help.
+
+## --build-dir=DIRECTORY-PATH
+
+Required.
+
+The directory where the tool will place its temporary files.
+
+## --image-file=FILE-PATH
+
+Required.
+
+The base image file to customize.
+
+Supported image file formats: vhd, vhdx, qcow2, and raw.
+
+## --output-image-file=FILE-PATH
+
+Required.
+
+The image format of the output customized image file.
```

Comment:
I expected the output format to be appended when I tried out the tool, but it was not. so, you may want to add an example here or mention that this should include the image extension.